app/main/presenters/service_presenters.py

```diff
@@ -24,14 +24,15 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
         if key[0] in service_data:
             setattr(self, key[1], service_data[key[0]])
 
-    def __init__(self, service_data, service_questions):
+    def __init__(self, service_data, service_questions, lots_by_slug):
         self.service_questions = service_questions
         # required attributes directly mapped to service_data values
         self.title = service_data['serviceName']
         self.serviceSummary = service_data['serviceSummary']\
             if 'serviceSummary' in service_data else \
             service_data['serviceDescription']
-        self.lot = service_data['lot']
+        # according to our test data, lot field in service_data may not be correct slug i.e. it may be mixed case
```

Comment:
@TheDoubleK have fixed up the tests now so we aren't using mixed-case lot slugs.